pkg/relayer/config/proxy_http_config_parser.go

```diff
@@ -48,6 +48,7 @@ func (supplierServiceConfig *RelayMinerSupplierServiceConfig) parseSupplierBacke
 	}
```

Comment:
Please update the configuration in this file with the new config: https://github.com/pokt-network/poktroll/blob/main/docusaurus/docs/operate/configs/relayminer_config.md

A markdown table w/ the currently support `Pocket-` headers would be super helpful